src/utils/cluster.ts

```diff
@@ -163,4 +165,10 @@ export class ClusterUtils {
       updateMask: {paths: this.getUpdateMask(metadata)},
     };
   }
+
+  static setBackupExpiryTime(backup: ModifiableBackupFields) {
+    if (backup.expireTime instanceof Date) {
```

Comment:
should we throw if the condition is not met here?

Reply:
No. This function is just pulling out the exact code that is being used in createBackup at https://github.com/googleapis/nodejs-bigtable/pull/1153/files#diff-9ce792749808a4cb9055a011d90ec2c35709e9deeee913947cb43f7ca394c911L317-L321 as this code is now used twice to parse a user provided date. backup.expireTime is of BackupTimestamp type and this means it can be a Date, PreciseDate or google.protobuf.ITimestamp as you can see by the definition in https://github.com/googleapis/nodejs-bigtable/blob/d30e5133a55e1b8f269b3aa6a69b49415361ba34/src/backup.ts#L44. Throwing an exception here would not let users provide a google.protobuf.ITimestamp value for their request.

Reply:
I think this method is misleading then. If I call `setBackupExpiryTime` and I pass in a non Date object (like an `ITimestamp` as you mention), there is no information to me that I am sending in something invalid, and nothing is getting set in that case. Can we rename the method to something more specific to the use case, or not extract it out?

Reply:
I changed the name to formatBackupExpiryTime. We should still extract it out so that we don't repeat code. Feel free to let me know if you have a different preferred name in mind.

Reply:
LGTM